translations/es/examples/index.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  title: 'Community Examples (Spanish)',
+  heading: `
+    If you've got an example of something that you've done with glamorous that
+    doesn't really work on one of the other pages, then please feel free to
+    open a pull request on this page!
+  `.replace(/~/g, '`'),
```

Comment:
It's replacing `~` with a backtick `\`` so we're not required to escape backticks in this snippet when we create codeblocks. It's hard to even type it in GitHub because they need to get escaped here too. 😀